bindings/nodejs/lib/types/models/api/validators-response.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Bech32Address, EpochIndex } from '../../block';
+import { u64 } from '../../utils';
+import type { HexEncodedString } from '../../utils/hex-encoding';
+
+/**
+ * Information of a validator.
+ */
+export interface ValidatorResponse {
+    /**
+     * Account address of the validator.
+     */
+    address: Bech32Address;
```

Comment:
I need you to remind me when/why to use `!` or not again. Why are you using it everywhere in `UtxoChangesResponse` for example and nowhere here?

Reply:
One can only use it in classes to show the compiler that the value can't be undefined or null
UtxoChangesResponse could also be an interface I guess, not sure what's better here

Reply:
So what's the rationale with `ValidatorResponse` being an interface and `UtxoChangesFullResponse` being a class for example? Are they used differently?

Reply:
`UtxoChangesFullResponse` contains outputs for which we want a class so one can call the output methods on it, the fields in `ValidatorResponse` you just access as they are, but you don't call any methods on them I would say

Reply:
Ok I see, thanks